db/seeds.rb

```diff
@@ -26,178 +26,178 @@
-item4 = Item.new(title: "Casserole",
-                 description: "Delicious casserole filled with bacon",
-                 price: 1200,
-                 image: open(path + "casserole.jpg"))
+#New York --------------------------------------------------------------------
+item4 = Item.new(title: "Big lights will inspire you",
+                 description: "City so nice they named it twice",
+                 price: 12000,
+                 image: open(path + "ex_new_york.jpg"))
 
 item4.categories << categories[1]
 item4.save
 
-item5 = Item.new(title: "Asparagus Mushroom Bacon Quiche",
-                 description: "Nothing better than a quiche with bacon",
-                 price: 1100,
-                 image: open(path + "quiche.jpg"))
+item5 = Item.new(title: "Empire State of Mind",
+                 description: "City so nice they named it thrice",
+                 price: 11000,
+                 image: open(path + "ex_new_york2.jpg"))
 
 item5.categories << categories[1]
 item5.save
 
-item6 = Item.new(title: "Grilled Cheese French Toast With Bacon",
-                 description: "Grilled cheese.  French toast.  Bacon.  Nuff said",
-                 price: 1500,
-                 image: open(path + "grilled_cheese.jpg"))
+item6 = Item.new(title: "These streets will make you feel brand new",
+                 description: "City so nice they named it one-time",
+                 price: 15000,
+                 image: open(path + "ex_new_york.jpg"))
 
 item6.categories << categories[1]
 item6.save
 
-#Lunch items----------------------------------------------------------------
+#Seattles ----------------------------------------------------------------
 
-item7 = Item.new(title: "Bacon-Wrapped Jalapeno Poppers",
-                 description: "Super spicy!",
-                 price: 1000,
-                 image: open(path + "pepper.jpg"))
+item7 = Item.new(title: "Rain City, USA Condo",
+                 description: "Cold, wet and smells like fish",
+                 price: 10000,
+                 image: open(path + "ex_seattle.jpg"))
 
 item7.categories << categories[2]
 item7.save
 
-item8 = Item.new(title: "Bacon Wrapped Chicken",
-                 description: "Farm raised chicken wrapped in bacon, and grilled to perfection",
-                 price: 1500,
-                 image: open(path + "bacon_wrapped_chicken.jpg"))
+item8 = Item.new(title: "Pike Place Market Bungalo",
+                 description: "Copper River Salmon!",
+                 price: 15000,
+                 image: open(path + "ex_seattle.jpg"))
 
 item8.categories << categories[2]
 item8.save
 
-item9 = Item.new(title: "Bacon Potato Soup",
-                description: "Hot soup with a touch of bacon",
-                price: 800,
-                image: open(path + "soup.jpg"))
+item9 = Item.new(title: "Mt Rainier Cabin",
+                description: "If you like camping on a volcano, we have the place for you!",
```

Comment:
Line is too long. [92/80]<br>Align the elements of a hash literal if they span more than one line.